packages/auth/src/config/app.ts

```diff
@@ -45,6 +45,8 @@ export const Config = {
     'AUTH_SERVER_DOMAIN',
     `http://localhost:${envInt('AUTH_PORT', 3006)}`
   ),
+  apiSecret: process.env.API_SECRET, // optional
+  apiSignatureVersion: envInt('API_SIGNATURE_VERSION', 1),
```

Comment:
Not related to this PR necessarily, but what is the benefit of having version? just to track how many times a key has been rotated, basically?

Also, should we call this "Admin API secret"?

Reply:
Not sure what the answer to having version is, but I'm down to rename the env var